stablehlo/reference/Tensor.h

```diff
@@ -46,15 +46,15 @@ class Buffer : public llvm::RefCountedBase<Buffer> {
   /// Returns type of the Buffer object.
   ShapedType getType() { return type_; }
 
-  /// Returns the raw data as a byte array.
-  char *getData() { return data_.data(); }
+  /// Provides access to the underlying non-mutable storage.
+  ArrayRef<char> getImmutableData() const { return blob_.getData(); }
```

Comment:
Any reason to change the name from `getData` to `getImmutableData`? Perhaps let's stick to whatever convention already exists upstream?